xcode/Subconscious/Shared/Components/AddressBook/MyDidView.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  MyDidView.swift
+//  Subconscious (iOS)
+//
+//  Created by Ben Follington on 3/3/2023.
+//
+
+import Foundation
+import SwiftUI
+
+struct MyDidView: View {
+    var myDid: Did
+    @State var didCopy = false
+    
+    var body: some View {
+        HStack{
+            Text(myDid.did)
+                .foregroundColor(.secondary)
+            Spacer()
+            ShareLink(item: myDid.did) {
+                Image(systemName: "square.and.arrow.up")
+            }
```

Comment:
I decided that using the share sheet is more idiomatic than a copy function for the DID, since you likely want to message it to someone via another app.